geoengine/ml.py

```diff
@@ -6,10 +6,14 @@
 import tempfile
 from typing import Protocol
 from dataclasses import dataclass
-from geoengine_openapi_client.models import MlModelMetadata, MlModel
+from onnx.reference import ReferenceEvaluator
+from onnx import TypeProto, TensorProto, ModelProto
```

Comment:
unrelated to this line: is it possible to show the interactive point labeling tool in the jupyter notebook (on github)? Right now in VSCode I get an error and in Githug there is just text.

If it's not possible maybe add a text that it's not shown here?

Reply:
ipywidgets are not shown on GitHub currently.

I tried to save it using a jupyter notebook viewer and https://ipywidgets.readthedocs.io/en/stable/embedding.html#embedding-widgets-in-the-sphinx-html-documentation

[c2b2bd6](https://github.com/geo-engine/geoengine-python/pull/196/commits/c2b2bd6d696f11ea72a1bb1e8e4cc9c0511c58fa)

Reply:
Does not seem to do much :shrug:

Reply:
ok then maybe write a line why nothing is visible or just resolve this comment if you dont want to do that.

Reply:
[77beb8a](https://github.com/geo-engine/geoengine-python/pull/196/commits/77beb8a47e17a691de20d06e290dc0bb2083d537)